lib/molecules/admin-bar/template.html

```diff
@@ -0,0 +1,25 @@
+<div class="toolbar" role="navigation" aria-label="Toolbar" tabindex="0" ng-if="vm.adminBarEnabled">
```

Comment:
Ok... but I think the sass should go here, not in ng-web-app. At least once we fix it so the ass can be built automatically.